packages/flutter_tools/lib/src/linux/build_linux.dart

```diff
@@ -114,6 +114,8 @@ Future<void> _runCmake(String buildModeName, Directory sourceDir, Directory buil
 Future<void> _runBuild(Directory buildDir) async {
   final Stopwatch sw = Stopwatch()..start();
 
+  final RegExp errorMatcher = RegExp(r'(?:.*:\d+:\d+|clang):\s?(?:error|warning):\s.*', caseSensitive: false);
```

Comment:
I would create this as a top level error. For bonus points, it would be great if you could document it a bit, and include some of examples too - regexs can be a bit hard to parse for humans

Reply:
err: top level variable.